### DISCUSSION TITLE
Setting q-btn styles for entire site

### DISCUSSION THREAD

**Discussion by maggie44** [quasarframework/quasar#11288] [General - Components / Directives / etc] [answered]
If I have a q-btn with some specific styles:

`<q-btn class="glossy" rounded color="deep-orange" label="Glossy Rounded" />`

And I want to use the same button type every time I insert a q-btn, is there a way to set these styles globally without having to include all the classes each time?

**Comment by hawkeye64** [comment]
@maggie0002 Create your own wrapper component around QBtn with the specific attributes/props set.

**Comment by pdanpdan** [comment, marked as answer]
Or make an object with all options you want as default and put `v-bind="defaultBtnConfig"` as first prop of QBtn (later props should override it)

https://codepen.io/pdanpdan/pen/abyaoPJ?editors=1010

**Reply by hawkeye64** [reply]
This is a better answer...
